ext/TransitionVisualizations.jl

```diff
@@ -0,0 +1,229 @@
+
+# Plot results of original input signal
+function TransitionsInTimeseries.plot_indicator_changes(res::SlidingWindowResults;
+        colors = default_colors,
+        indicator_names = default_indicator_label(res),
+        chametric_names = default_chametric_label(res),
+        additional_timeseries = nothing,
+        accent_linewidth = default_accent_linewidth,
+        kwargs...,
+    )
+
+    fig, laxs, raxs, n, config = init_rowwise_visualisation(res, colors, indicator_names,
+        chametric_names, additional_timeseries, accent_linewidth)
+    lineplot_metrics!(raxs, laxs, n, config, res.t_indicator, res.x_indicator,
+        res.t_change, res.x_change, colors, accent_linewidth)
+
+    return TransitionVisualization(fig, laxs, raxs, res, colors)
+end
+
+function TransitionsInTimeseries.plot_indicator_changes(res::SegmentedWindowResults;
+    colors = default_colors,
+    indicator_names = default_indicator_label(res),
+    chametric_names = default_chametric_label(res),
+    additional_timeseries = nothing,
+    accent_linewidth = default_accent_linewidth,
+    kwargs...,
+    )
+
+    fig, laxs, raxs, n, config = init_rowwise_visualisation(res, colors, indicator_names,
+        chametric_names, additional_timeseries, accent_linewidth)
+    for k in eachindex(res.t_indicator)
+        Makie.lines!(laxs[1], res.t[k], res.x[k], color = colors[1],
+            linewidth = accent_linewidth)
+        lineplot_metrics!(raxs, laxs, n, config, res.t_indicator[k], res.x_indicator[k],
+            res.t_change[k], res.x_change[k, :], colors, accent_linewidth)
+    end
+
+    return TransitionVisualization(fig, laxs, raxs, res, colors)
+end
+
+# utils for plot_indicator_changes
+function init_rowwise_visualisation(res, colors, indicator_names,
+    chametric_names, additional_timeseries, accent_linewidth)
+
+    config = res.config
+    fig = Makie.Figure(size = (700, 450), fontsize = 12)
+    rlabels = vcat([""], indicator_names)
+    llabels = vcat(["Input"], chametric_names)
+    n = length(llabels)
+
+    rowaspect = 5
+    raxs = [Makie.Axis(fig[(i-1)*rowaspect+1:i*rowaspect, 1], ylabel = rlabels[i],
+        xticklabelsvisible = false, yaxisposition = :right, ygridvisible = false,
+        ylabelcolor = colors[2], yticklabelcolor = colors[2]) for i in 1:n]
+    laxs = [Makie.Axis(fig[(i-1)*rowaspect+1:i*rowaspect, 1], ylabel = llabels[i],
+        xticklabelsvisible = false, ygridvisible = false,
+        ylabelcolor = colors[1], yticklabelcolor = colors[1]) for i in 1:n]
+    Makie.linkxaxes!(laxs..., raxs...)
+
+    hidedecorations!(raxs[1])
+    Makie.lines!(laxs[1], res.t, res.x, color = colors[1], linewidth = accent_linewidth)
+
+    raxs[end].xticklabelsvisible = true
+    raxs[end].xlabel = "Time"
+    Makie.rowgap!(fig.layout, 10)
+
+    elements = [LineElement(color = (colors[1], transparency), linewidth = lw) for 
+        (lw, transparency) in [(accent_linewidth, 1), (1, 0.5)]]
+    labels = ["original signal", "surro signals"]
```

Comment:
I think we don't need to overplot labels at the top of the figure. The docstring should make it clear that surrogates are plotted as thinner lines and that should be enough...?

Reply:
I think it is helpful to have it there, without presenting any major drawback